resources/views/layouts/footer.blade.php

```diff
@@ -0,0 +1,67 @@
+<div class="container text-white">
+  <div class="row">
+      <div class="col-3">
+        <a class="navbar-brand" href="{{ url('/') }}">
+        <img src="/images/logo.png" height="50px" class="mt-3 ms-3">
```

Comment:
can you do it like this? for the img src
{{ asset('images/logo.png') }}

Reply:
I changed the img src like {{ asset('images/logo.png') }}